Lighthouse Camera/UI/Main/AlertHelper.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AlertHelper.swift
+//  Lighthouse Camera
+//
+//  Created by Punam Dahiya on 1/26/17.
+//  Copyright © 2017 Lighthouse. All rights reserved.
+//
+
+extension UIAlertController {
+  // Customise UIAlertController to handle permission failure alerts
+
+  enum AlertMessageID {
+    case CameraFailed, MicrophoneFailed
```

Comment:
Since you're asking me for a review, I'll ask you for more documentation :)
In this case, who's expected to send `CameraFailed`, `MicrophoneFailed`? And can you mention *what* failed?